packages/roosterjs-editor-plugins/test/TableResize/tableResizeTest.ts

```diff
@@ -0,0 +1,135 @@
+    });
+
+    type Position = {
+        x: number;
+        y: number;
+    };
+
+    function initialize(isRtl: boolean = false): DOMRect {
+        if (isRtl) {
+            editor.getDocument().body.style.direction = 'rtl';
+        }
+        editor.setContent(TABLE);
+        let tableRect: DOMRect = null;
+        editor.queryElements('table', table => {
+            const rect = table.getBoundingClientRect();
+            tableRect = rect;
+        });
+        return tableRect;
+    }
+
+    function runTest(mouseStart: Position, mouseEnd: Position, expectedResult: boolean) {
+        const editorDiv = editor.getDocument().getElementById(TEST_ID);
+
+        let mouseEvent = new MouseEvent('mousemove', {
+            clientX: mouseStart.x,
+            clientY: mouseStart.y,
+        });
+
+        editorDiv.dispatchEvent(mouseEvent);
+
+        const body = editor.getDocument().body;
+
+        expect(body.innerHTML.includes(ADD_BUTTON)).toBe(true);
+
+        mouseEvent = new MouseEvent('mousemove', {
+            clientX: mouseEnd.x,
+            clientY: mouseEnd.y,
+        });
+        editorDiv.dispatchEvent(mouseEvent);
+        expect(body.innerHTML.includes(ADD_BUTTON)).toBe(expectedResult);
+    }
+
+    it('removes the vertical inserter when moving the cursor out of the offset zone', () => {
+        const rect = initialize();
+        runTest({ x: rect.right, y: rect.top }, { x: rect.right / 2, y: rect.bottom }, false);
+    });
+
+    it('keeps the vertical inserter when moving the cursor inside the safe zone', () => {
+        const rect = initialize();
+        runTest(
+            { x: rect.right, y: rect.top },
+            { x: rect.right - insideTheOffset, y: rect.top },
+            true
+        );
+    });
+
+    it('removes the horizontal inserter when moving the cursor out of the offset zone', () => {
+        const rect = initialize();
+        runTest(
+            { x: rect.left, y: rect.bottom },
+            { x: (rect.right - rect.left) / 2, y: (rect.bottom - rect.top) / 2 },
+            false
+        );
+    });
+
+    it('keeps the horizontal inserter when moving the cursor inside the safe zone', () => {
+        const rect = initialize();
+        runTest(
+            { x: rect.left, y: rect.bottom },
+            { x: rect.left, y: rect.bottom - insideTheOffset },
+            true
+        );
+    });
+
+    it('removes the vertical inserter when moving the cursor out of the offset zone with culture language RTL', () => {
+        const rect = initialize(true);
+        runTest(
+            { x: rect.left, y: rect.top },
+            { x: (rect.right - rect.left) / 2, y: rect.bottom },
+            false
+        );
+    });
+
+    it('keeps the vertical inserter when moving the cursor inside the safe zone with culture language RTL', () => {
+        const rect = initialize(true);
+        runTest(
+            { x: rect.left, y: rect.top },
```

Comment:
It seems that it won't be able to display the vertical inserter if we land the cursor exactly on the left border, the pixel might be too small to be detected in that scenario. 

![notshown](https://user-images.githubusercontent.com/4762468/125149153-1ab67180-e0ec-11eb-81f8-c32103cb38cb.jpg)

let's not land the cursor exactly on the left border, something like below can fix it.

{ x: rect.left + insideTheOffset, y: rect.top }